framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -532,10 +668,14 @@ public Void visitTypeCast(TypeCastTree tree, AnnotatedTypeMirror type) {
         private List<?> getValues(AnnotatedTypeMirror type, TypeMirror castTo) {
             AnnotationMirror anno = type.getAnnotationInHierarchy(UNKNOWNVAL);
             if (anno == null) {
-                // if type is an AnnotatedTypeVariable (or other type without a primary annotation)
-                // then anno will be null. It would be safe to use the annotation on the upper bound;
-                //  however, unless the upper bound was explicitly annotated, it will be unknown.
-                // AnnotatedTypes.findEffectiveAnnotationInHierarchy(, toSearch, top)
+                // if type is an AnnotatedTypeVariable (or other type without a
```

Comment:
This is a gratuitous whitespace change.  The whole file is wrapped to 100 characters, and this comment used to be.  Therefore, I think it's a bad idea to add a newline to every line.